osu.Game/Overlays/FirstRunSetup/ScreenImportFromStable.cs

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Localisation;
+using osu.Game.Database;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.UserInterfaceV2;
+using osu.Game.Localisation;
+using osu.Game.Overlays.Settings;
+using osu.Game.Screens.Edit.Setup;
+using osuTK;
+
+namespace osu.Game.Overlays.FirstRunSetup
+{
+    [LocalisableDescription(typeof(FirstRunOverlayImportFromStableScreenStrings), nameof(FirstRunOverlayImportFromStableScreenStrings.Header))]
+    public class ScreenImportFromStable : FirstRunSetupScreen
+    {
+        private static readonly Vector2 button_size = new Vector2(400, 50);
+
+        private ProgressRoundedButton importButton = null!;
+
+        private OsuTextFlowContainer progressText = null!;
+
+        [Resolved]
+        private LegacyImportManager legacyImportManager { get; set; } = null!;
+
+        private StableLocatorLabelledTextBox stableLocatorTextBox = null!;
+
+        private IEnumerable<ImportCheckbox> contentCheckboxes => Content.Children.OfType<ImportCheckbox>();
+
+        [BackgroundDependencyLoader(permitNulls: true)]
+        private void load()
+        {
+            Content.Children = new Drawable[]
+            {
+                new OsuTextFlowContainer(cp => cp.Font = OsuFont.Default.With(size: CONTENT_FONT_SIZE))
+                {
+                    Colour = OverlayColourProvider.Content1,
+                    Text = FirstRunOverlayImportFromStableScreenStrings.Description,
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y
+                },
+                stableLocatorTextBox = new StableLocatorLabelledTextBox
+                {
+                    Label = FirstRunOverlayImportFromStableScreenStrings.LocateDirectoryLabel,
+                    PlaceholderText = FirstRunOverlayImportFromStableScreenStrings.LocateDirectoryPlaceholder
+                },
+                new ImportCheckbox("Beatmaps", StableContent.Beatmaps),
+                new ImportCheckbox("Scores", StableContent.Scores),
+                new ImportCheckbox("Skins", StableContent.Skins),
+                new ImportCheckbox("Collections", StableContent.Collections),
```

Comment:
Unlocalised strings here (and the class looks to support localisation) - oversight?

Reply:
A bit of an oversight yeah. Was on my list but wanted to look up to see if these are already translated in strings (which is non-zero-effort right now).